src/lib/meilisearch.ts

```diff
@@ -253,14 +256,68 @@ class MeiliSearch {
   ///
 
   /**
-   * Get private and public key
+   * Get all API keys
    * @memberof MeiliSearch
    * @method getKey
-   * @returns {Promise<Keys>} Promise containing an object with keys
+   * @returns {Promise<Keys>} Promise returning an object with keys
    */
-  async getKeys(): Promise<Keys> {
+  // FIXME: should be Result<Key[]>>
+  async getKeys(): Promise<Key[]> {
     const url = `keys`
-    return await this.httpRequest.get<Keys>(url)
+    return await this.httpRequest.get<Key[]>(url)
+  }
+
+  /**
+   * Get one API key
+   * @memberof MeiliSearch
+   * @method getKey
+   *
+   * @param {string} key - Key
+   * @returns {Promise<Keys>} Promise returning a key
```

Comment:
```suggestion
   * @returns {Promise<Key>} Promise returning a key
```

Reply:
More generally sometimes you use:
`Promise returning a key`
and others:
`Promise returning an object with keys`
Not sure to understand the difference

Reply:
in this key it is because it is `one key`. In the above example it is because it is supposed to be an object like that: 

```
{ 
  results: [...]
}
```